src/phantasm-renderer/backend/detail/hash.hh

```diff
@@ -1,10 +1,12 @@
 #pragma once
 
+#include <functional>

```

Comment:
Why do we need `functional` here? it's one of the biggest std headers and I don't see where it is used

Reply:
Sadly we need it for `std::hash`, at least with clang